prac_08/dynamic_labels.kv

```diff
@@ -0,0 +1,6 @@
+BoxLayout:
+    BoxLayout:
+        orientation: 'vertical'
+        id: entries_box
+    Label:
+        text: app.status_text
```

Comment:
I think it would be a nicer GUI if you delete line 5 and 6. Your not using this label in the python file as far as i can see.